src/packages/cli/package.json

```diff
@@ -55,6 +55,7 @@
     "@ganache/colors": "^0.1.0",
     "@ganache/ethereum": "0.1.0",
     "@ganache/ethereum-options": "0.1.0",
+    "@ganache/filecoin": "0.1.0",
```

Comment:
> This gets into the "fancy downloader" convo, so maybe it's time we have that.

Because of this, I'm favoring that we leave this as is *until* we're actually considering merging this into `develop`. That way I can continue working without any fancy stuff. We can determine the delivery mechanism and how I need to support that later. I don't think we should work any of this part until we're closer to merging this into `develop` (probably early March)

> and the version should probably be `"^0.1.0"`

When should versions be pinned and when shouldn't they be? This just did whatever the default `lerna add` behavior was, but I am trying to pin everything new I add due to [the contribution guidelines](https://github.com/trufflesuite/ganache-core/blob/develop/CONTRIBUTING.md#L164)

Reply:
FYI I went ahead and added initial support for this as it's necessary for some of my future PRs to be acceptable for you (otherwise I'd have to turn off all tests with node versions less than 12 rather than just filecoin tests)

I added the logic in `@ganache/flavors`. I initially was thinking of putting it in `connector-loader.ts` like you, but something felt better about putting it flavors. We can definitely discuss more whenever we review that work